ui.R

```diff
@@ -172,7 +172,26 @@ div.myspecial-popup div.leaflet-popup-content-wrapper {
       )
     )
   ),
-  dfeshiny::header(site_title),
+  # Force the top nav bar to left align and centre the title
+  HTML(
+    '<header class="govuk-header" role="banner">
+    <div class="govuk-header__container">
+    <div class="govuk-header__logo" style="width: 15%; margin-left: 15px;float:left;">
+    <a href="https://www.gov.uk/government/organisations/department-for-education" class="govuk-header__link govuk-header__link--homepage">
+    <span class="govuk-header__logotype">
+   <img src="images/DfE_logo.png" class="govuk-header__logotype-crown-fallback-image"/>
+    <span class="govuk-header__logotype-text">DfE</span>
```

Comment:
Looks a bit odd having DfE in non standard font. It'd be good to remove this and just use the standard logo image which has the DfE name in the standard font.